<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="loadContactsList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>520</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить список контактов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadStatistic">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>520</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить статистику клиентов</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="Statistic">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>381</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата/Время</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ip</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="contactsList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
